server/dataset/cxg_dataset.py

```diff
@@ -393,7 +427,13 @@ def _get_schema(self):
         annotations = {}
         for ax in ("obs", "var"):
             A = self.open_array(ax)
-            schema_hints = json.loads(A.meta["cxg_schema"]) if "cxg_schema" in A.meta else {}
+
+            if ax == "obs":
+                # Filter out int64 values in the obs cxg_schema metadata
+                schema_hints = self._get_filtered_cxg_schema_metadata()
```

Comment:
same issue here with opening up the `obs` array twice

Reply:
https://github.com/chanzuckerberg/single-cell-explorer/blob/025beb186f5d2f718ec6cf7446032e3b59d6aa1c/server/dataset/cxg_dataset.py#L198

not sure what you're referring to, but if it's this function "open_array", then it's not opening multiple connections to TileDB. Just returning the same already opened connection from a hash table.

Reply:
awesome thanks!